apps/frontend/src/components/fap/MeetingComponents/FapUpdateInstrumentTime.tsx

```diff
@@ -0,0 +1,89 @@
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogTitle from '@mui/material/DialogTitle';
+import TextField from '@mui/material/TextField';
+import makeStyles from '@mui/styles/makeStyles';
+import { Form, Formik } from 'formik';
+import React, { ChangeEvent, useState } from 'react';
+
+import { InstrumentWithAvailabilityTime } from 'generated/sdk';
+import useDataApiWithFeedback from 'utils/useDataApiWithFeedback';
+
+const useStyles = makeStyles((theme) => ({
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+  },
+}));
+
+const MAX_32_BIT_INTEGER = Math.pow(2, 31);
+
+type FapUpdateInstrumentTimeProps = {
+  close: () => void;
+  updateTime: (newTime: number, instrumentId: number) => void;
+  callId: number;
+  instrument: InstrumentWithAvailabilityTime;
+};
+
+const AvailabilityTimeEditComponent = (
+  instTime: string | undefined,
+  onChange: (newTime: string) => void
+) => (
+  <TextField
+    type="number"
+    data-cy="availability-time"
+    value={instTime}
+    onChange={(e: ChangeEvent<HTMLInputElement>) => onChange(e.target.value)}
+    InputProps={{ inputProps: { max: MAX_32_BIT_INTEGER - 1, min: 0 } }}
+    required
+    fullWidth
+    margin="none"
+  />
+);
+
+const FapUpdateInstrumentTime = (props: FapUpdateInstrumentTimeProps) => {
+  const classes = useStyles();
+  const { api } = useDataApiWithFeedback();
+  const [newTime, SetNewTime] = useState<string | undefined>(
+    props.instrument.availabilityTime?.toString()
+  );
+
+  const updateTimeAvailable = () => {
+    newTime &&
+      api({
+        toastSuccessMessage: 'Availability time updated successfully!',
+      }).setInstrumentAvailabilityTime({
+        callId: props.callId,
+        instrumentId: props.instrument.id,
+        availabilityTime: +newTime,
+      });
+  };
+
+  return (
+    <Dialog open={true} onClose={props.close}>
+      <DialogTitle variant="h6" component="h1">
```

Comment:
I think it would make more sense for the dialog title to be in a header tag numbered higher than the one used for the title of the FAP (`h2`). Ideally the latter would be `h1` and the former `h2`.